src/api/public/apidocs-new/paths/source_project_name_attribute_attribute_name.yaml

```
@@ -5,30 +5,47 @@ get:
   parameters:
     - $ref: '../components/parameters/project_name.yaml'
     - $ref: '../components/parameters/attribute_name.yaml'
-    - name: meta
+    # Query Strings:
+    # `rev`: doesn't make sense in project level and makes the endpoint crash.
+    # `meta`: either you pass `meta` or not, it's always replaced by meta: 1.
+    # `with_project`: does not make sense in project level and makes the endpoint crash.
+    - name: view
       in: query
       schema:
         type: string
-    - name: rev
+      description: With view=blame, information about who changed each XML tag and when is displayed.
+      enum:
+        - blame
```

```suggestion
        enum:
          - blame
      description: With view=blame, information about who changed each XML tag and when is displayed.
```
The `enum:` key should belong to the `schema:` key.